### ISSUE TITLE
opam install fails

### ISSUE THREAD

**Issue by svandragt** [savonet/liquidsoap#535]
All decencies are met, but install fails:
```
$ opam install liquidsoap
The following actions will be performed:
  ∗  install liquidsoap 1.3.3

=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[liquidsoap] Archive in cache

=-=- Processing actions -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ERROR] The compilation of liquidsoap failed at "make".
Processing  1/1: [liquidsoap: rm]
#=== ERROR while installing liquidsoap.1.3.3 ==================================#
# opam-version 1.2.2
# os           darwin
# command      make
# path         /Users/svandragt/.opam/system/build/liquidsoap.1.3.3
# compiler     system (4.06.1)
# exit-code    2
# env-file     /Users/svandragt/.opam/system/build/liquidsoap.1.3.3/liquidsoap-86067-e7966f.env
# stdout-file  /Users/svandragt/.opam/system/build/liquidsoap.1.3.3/liquidsoap-86067-e7966f.out
# stderr-file  /Users/svandragt/.opam/system/build/liquidsoap.1.3.3/liquidsoap-86067-e7966f.err
### stdout ###
# if test -e ../.git && which git > /dev/null ; then \
# [...]
#           date=`date +%Y%m%d:%H%M%S`; \
# 	  echo "let rev = \" ($branch@$revision:$date)\"" > SVN.ml ; \
# 	else \
# 	  echo "let rev = \"\"" > SVN.ml ; \
# 	fi
# OCAMLDEP
# OCAMLOPT -c shutdown.ml
# OCAMLOPT -c tools/stdlib.ml
# OCAMLOPT -c tools/doc.ml
### stderr ###
# File "tools/doc.ml", line 62, characters 42-48:
# Error: This expression has type bytes but an expression was expected of type
#          string
# make[2]: *** [tools/doc.cmx] Error 2
# make[1]: *** [all-auto-ocaml-prog] Error 2
# make: *** [all-subdirs] Error 2



=-=- Error report -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
The following actions failed
  ∗  install liquidsoap 1.3.3
No changes have been performed

````
I also can't install vorbis so I omitted that (https://github.com/savonet/ocaml-vorbis/issues/1) in case it's related to this error. Any idea? I'm not familiar with ocam but happy to provide further information.

macOS 10.13.3

**Comment by janvladimirmostert** [comment]
Related: https://github.com/savonet/liquidsoap/issues/538

**Comment by toots** [comment]
Should be fixed with `1.3.3-1` Please note that you need `OCaml <= 4.05` until the next release.
